src/xenia/app/emulator_window.cc

```diff
@@ -1382,6 +1394,21 @@ void EmulatorWindow::ToggleDisplayConfigDialog() {
   }
 }
 
+void EmulatorWindow::ToggleProfilesConfigDialog() {
+  if (!profile_config_dialog_) {
+    disable_hotkeys_ = true;
+    emulator_->kernel_state()->BroadcastNotification(kXNotificationIDSystemUI,
```

Comment:
You need to also increment/decrement xam_dialogs_shown_ var so that XamIsUIActive works.